src/core/middleware/security.js

```diff
@@ -37,7 +37,7 @@ export function csp({ _config = config, noScriptStyles, _log = log } = {}) {
     _config.get('CSP') !== 'false' ? deepcopy(_config.get('CSP')) : false;
 
   if (cspConfig) {
-    if (noScriptStyles) {
+    if (noScriptStyles && !_config.get('isDevelopment')) {
```

Comment:
I think it would be good to add [a test for this](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/core/middleware/test_cspMiddleware.js).

Reply:
ah, right